deps.windows/60-uthash.ps1

```diff
@@ -0,0 +1,56 @@
+param(
+    [string] $Name = 'uthash',
+    [string] $Version = '2.3.0',
+    [string] $Uri = 'https://github.com/troydhanson/uthash.git',
+    [string] $Hash = "e493aa90a2833b4655927598f169c31cfcdf7861"
+)
+
+function Setup {
+    Setup-Dependency -Uri $Uri -Hash $Hash -DestinationPath $Path
+}
+
+function Install {
+    Log-Information "Install (${Target})"
+    Set-Location "$Path"
+
+    New-Item -Path "$($ConfigData.OutputPath)/include" -ItemType Directory -Force *> $null
+
+    $Items = @(
+        @{
+            Path = "src/utarray.h"
```

Comment:
You should be able to specify an array here (`"src/utarray.h", "src/uthash.h", [..]`), PowerShell usually accepts that as input parameter for file operations.

Reply:
No log is fine, or do you want some kind of logging ?

I ask before I resolve this and squash my commits.

Reply:
Why did you remove the rest of the code? That code was already compatible with providing an array (hence why it did the whole thing about joining the string). The only change necessary was to have a single item with the files provided as an array. :D

Reply:
You say to use an array which usually make the whole for loop useless (and so the log per file).

Reply:
So you want to keep the loop ?